pom.xml

```diff
@@ -5,9 +5,10 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.epam.reportportal</groupId>
     <artifactId>agent-java-junit</artifactId>
-    <version>4.0.0</version>
+    <version>4.0.2-SNAPSHOT</version>

```

Comment:
i would make it 4.1 since we drop JDK7 support

Reply:
The support for JDK 7 was dropped at version 4.0.0, but other changes in this PR warrant a minor-version bump.